go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	golang.org/x/net v0.0.0-20210503060351-7fd8e65b6420
 	golang.org/x/oauth2 v0.0.0-20210514164344-f6687ab2804c
 	golang.org/x/perf v0.0.0-20211012211434-03971e389cd3
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
```

Comment:
This is not needed now, right?

Reply:
Updated to go 1.17.
+ "go mod tidy"
+ "go mod vendor"
Got this result. This means it is needed.

Reply:
I can't find any x/sync.Mutex uses anywhere, but for Mutex use the standard library sync.Mutex.

Reply:
"go clean --modcache" was needed additionally.